packages/drivers/odsp-driver-definitions/src/odspCache.ts

```diff
@@ -127,5 +127,10 @@ export interface IPersistedCache {
  * @internal
  */
 export function getKeyForCacheEntry(entry: ICacheEntry): string {
-	return `${entry.file.docId}_${entry.type}_${entry.key}`;
+	const version =
```

Comment:
We have this same function in driver-web-cache as well which seems to be actuallty getting used to create the key for the db storage.